products/dns/terraform.yaml

```diff
@@ -0,0 +1,66 @@
+# Copyright 2017 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Provider::Terraform::Config
+overrides: !ruby/object:Provider::ResourceOverrides
+  ManagedZone: !ruby/object:Provider::Terraform::ResourceOverride
+    exclude: false
+    id_format: "{{name}}"
+    import_format: ["projects/{{project}}/managedZones/{{name}}"]
+    description: |
```

Comment:
Instead of overriding the description, let's add the API links to the api.yaml file so all the providers have access to them. (there are a bunch of examples in compute/api.yaml)